unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/StickersController/StickersController.cs

```diff
@@ -1,20 +1,29 @@
-﻿using UnityEngine;
+using UnityEngine;
 
 public class StickersController : MonoBehaviour
 {
     private StickersFactory stickersFactory;
 
     private void Awake() { stickersFactory = Resources.Load<StickersFactory>("StickersFactory"); }
 
-    public void PlayEmote(string id)
+    public void PlaySticker(string id)
+    {
+        PlaySticker(id, transform.position, Vector3.zero, true);
+    }
+
+    public void PlaySticker(string id, Vector3 position, Vector3 direction, bool followTransform)
     {
         if (stickersFactory == null || !stickersFactory.TryGet(id, out GameObject prefab))
             return;
 
         GameObject emoteGameObject = Instantiate(prefab);
-        emoteGameObject.transform.position += transform.position;
-        FollowObject emoteFollow = emoteGameObject.AddComponent<FollowObject>();
-        emoteFollow.target = transform;
-        emoteFollow.offset = prefab.transform.position;
+        emoteGameObject.transform.position += position;
+        emoteGameObject.transform.rotation = Quaternion.Euler(prefab.transform.rotation.eulerAngles + direction);
+
+        if (followTransform) {
```

Comment:
Please leave a new line before the brackets.
```suggestion
        if (followTransform) 
        {
```